src/scss/App.scss

```diff
@@ -0,0 +1,76 @@
+@import '_variables.scss';
+@import './libs/normalize';
+@import './components/all';
+
+
+.app{
+  display: flex;
+  &__footer{
+    display: flex;
+    flex-direction: row;
+  position: absolute;
```

Comment:
what happens to ident here ?

Reply:
such an design, the footer is in the bottom left corner, absolute